src/index.ts

```diff
@@ -37,34 +50,73 @@ export const runScenario = (
             );
             process.exit(1);
         }
-        getLogger('unknown').info(
-            `Starting scenario: ${scenarioPath} (actions: ${actionDir}, out: ${outDir}, envConfig: ${envConfigFile})`,
+
+        getLogger('setup').info(
+            `RUNNING: scenario(s): ${scenarioPaths} (actions: ${actionDir}, out: ${outDir}, envDir: ${envConfigDir}, numberOfScenariosRunInParallel: ${numberOfScenariosRunInParallel}, environmentNameToBeUsed: ${environmentNameToBeUsed})`,
         );
 
         OUT_DIR = outDir;
 
-        const envConfig = envConfigFile
-            ? loadYamlConfiguration(envConfigFile)
+        const envConfig = envConfigDir
+            ? loadYamlConfiguration(
+                  `${envConfigDir}/${environmentNameToBeUsed}.yaml`,
+              )
             : {};
+        getLogger('setup').debug(
+            `Using '${environmentNameToBeUsed}' configuration: ${stringify(
+                envConfig,
+            )}`,
+        );
 
         const actions: Action[] = loadAllActions(actionDir, envConfig);
+        getLogger('setup').debug(
+            `Successfully loaded ${actions.length} actions`,
+        );
 
-        const scenarios: Scenario[] = scenarioPath.endsWith('yaml')
-            ? loadScenariosById(scenarioPath, actions)
-            : loadAllScenarios(scenarioPath, actions);
+        scenarioPaths.forEach(scenarioPath => {
+            const scenario: Scenario[] = scenarioPath.endsWith('yaml')
```

Comment:
should probably be caled `scenarios` instead.

Reply:
yes!